src/dev/ci_setup/setup.sh

```diff
@@ -29,7 +29,7 @@ fi
 ### install dependencies
 ###
 echo " -- installing node.js dependencies"
-yarn kbn bootstrap --verbose
+yarn kbn bootstrap --force-install --verbose
```

Comment:
Why do we need force-install here?

Reply:
At the beginning I wasn't use it but it looks like due to the way we cache node_modules without `--force-install` the yarn installation will not be triggered (probably because `.yarn-integrity` is present)

Reply:
Wouldn't that be a similar issue with local development when updating a branch?

Reply:
I am wondering if we need to wait until everything is migrated in order to change the default.

Reply:
I was able to work around that issue 😃